web/package.json

```diff
@@ -39,9 +39,9 @@
     "lint": "eslint .",
     "server": "webpack-dev-server --inline --hot",
```

Comment:
Cleaning out the repo, installing the dependencies and running a fresh build (`git clean -xdf && npm install && npm run build`) works as usual. 
But when I try to spin up the dev server (`npm run start:web`), I get this error:

```
@batch/explorer-web: > webpack-dev-server --inline --hot "--env" "dev"
@azure/bonito-core: 4:04:42 PM - Found 0 errors. Watching for file changes.
@batch/explorer-web: [webpack-cli] Error: Unknown option '--inline'
@batch/explorer-web: [webpack-cli] Run 'webpack --help' to see available commands and options
```

Reply:
Thanks - not sure how I didn't catch this earlier. Pushed a fix.

Since I was already making changes to the webpack dev server configuration, I also switched it so the web package puts localized strings in a top-level resources folder just like the other projects (and webpack-dev-server serves files out of that folder).